src/encoded/static/components/browse/MicroscopySearchView.js

```diff
@@ -0,0 +1,263 @@
+'use strict';
+
+import React from 'react';
+import memoize from 'memoize-one';
+import _ from 'underscore';
+import url from 'url';
+import { DropdownItem, DropdownButton, Modal } from 'react-bootstrap';
+import { getAbstractTypeForType } from '@hms-dbmi-bgm/shared-portal-components/es/components/util/schema-transforms';
+import { SearchView as CommonSearchView } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/SearchView';
+import { console, isSelectAction, ajax, navigate , object } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { Alerts } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/Alerts';
+import { columnExtensionMap } from './columnExtensionMap';
+import { memoizedUrlParse } from './../globals';
+import { Schemas } from './../util';
+
+
+const microscopyColExtensionMap = _.extend({}, columnExtensionMap, {
+    "microscope.Tier": { "widthMap": { "sm": 50, "md": 70, "lg": 80 } },
+    "microscope.Manufacturer": { "widthMap": { "sm": 80, "md": 100, "lg": 120 } },
+    "microscope.Serial": { "widthMap": { "sm": 80, "md": 100, "lg": 120 } },
+    "microscope.Model": { "widthMap": { "sm": 80, "md": 100, "lg": 120 } },
+    "microscope.Type": { "widthMap": { "sm": 80, "md": 100, "lg": 120 } },
+    "submitted_by.display_title": { "widthMap": { "sm": 100, "md": 100, "lg": 120 } },
+});
+export default class MicroscopySearchView extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.handleModalCancel = this.handleModalCancel.bind(this);
+        this.handleConfirm = _.throttle(this.handleConfirm.bind(this), 3000);
+        this.handleMicroscopeNameChange = this.handleMicroscopeNameChange.bind(this);
+
+        this.state = {
+            show: false,
+            tier: 1,
+            validationTier: 1,
+            microscopeName: '',
+            cloneLoading: false
+        };
+
+        this.searchResultTableRef = React.createRef();
+    }
+    leftButtonDropDownButton() {
+        const { show, tier, validationTier, microscopeName,cloneLoading } = this.state;
+        const validationTierOptions=_.range(1,tier+1);
+        return (
+            <>
+                <DropdownButton id="search-item-type-selector" onSelect={(eventKey, evt) => { this.onChangeCreateTier(eventKey); }}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSX props should not use arrow functions (react/jsx-no-bind)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)